.travis.yml

```diff
@@ -3,6 +3,7 @@ python:
 - 2.7
 install:
 - pip install -r requirements.txt
```

Comment:
If requirements-dev.txt has the content specified in https://github.com/napalm-automation/napalm-ios/issues/22, can remove this line